src/components/IncidentForm/IncidentForm.js

```diff
@@ -12,24 +13,24 @@ const IncidentForm = () => {
   const [incidentDate, setIncidentDate] = useState('');
   const [incidentSummary, setIncidentSummary] = useState('');
   const [incidentLocation, setIncidentLocation] = useState('');
+  const [incidentCity, setIncidentCity] = useState('');
+  const [incidentState, setIncidentState] = useState('');
 
   const dispatch = useDispatch();
 
   const handleSubmit = e => {
     e.preventDefault()
     const incidentObj = {
-      id: `${Date.now()}`,
-      type: 'incident',
-      attributes: {
-        name: incidentName,
-        type: incidentType,
-        status: 'open',
-        description: incidentSummary,
-        location: incidentLocation,
-        start_date: incidentDate,
-        close_date: ""
-      }
+      name: incidentName,
+      incident_type: incidentType,
+      description: incidentSummary,
+      location: incidentLocation,
+      city: incidentCity,
+      state: incidentState,
+      start_date: incidentDate,
+      close_date: ""
     }
```

Comment:
Could've sworn I reformatted the body for this POST request in a previous branch. Regardless, glad we were able to get this fixed. Nice to see our POST requests working now for new incident and new contact.